build/docker-compose.yaml

```diff
@@ -2,14 +2,14 @@ version: "3"
 
 services:
   common: &common
-    image: foundationdb/foundationdb-build:0.0.5
+    image: foundationdb/foundationdb-build:0.0.7
 
   build-setup: &build-setup
     <<: *common
     depends_on: [common]
     volumes:
-      - ..:/foundationdb
-    working_dir: /foundationdb
+      - ..:/__this_is_some_very_long_name_dir_that_is_needed_to_fix_a_bug_with_cpack__/foundationdb
```

Comment:
small correction: this is not a bug in cpack but an issue with RPM debug packages. But I don't think it matters too much